src/splitInteger.test.js

```diff
@@ -4,18 +4,36 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(8, 1)).toEqual([8]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(6, 2)).toEqual([3, 3]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
-
+  expect(splitInteger(17, 4)).toEqual([4, 4, 4, 5]);
 });
 
 test('should add zeros if value < numberOfParts', () => {
+  expect(splitInteger(32, 6)).toEqual([5, 5, 5, 5, 6, 6]);
+});
 
+test(`should split a number into equal parts
+  if a value is divisible by a numberOfParts`, () => {
```

Comment:
should split a number into equal parts
  if a value is divisible by a numberOfParts (e.g. 15, 3) [5, 5, 5]